src/templates/blog-list-template.js

```diff
@@ -2,16 +2,26 @@ import React from 'react'
 import { graphql } from 'gatsby'
 import Layout from '~components/layout'
 import BlogTeaserList from '~components/pages/blog/blog-teaser-list'
+import BlogNavigation from '~components/pages/blog/blog-pagination'
```

Comment:
Can we keep either the `BlogNavigation` or the `BlogPagination` name, but not both?
Let's either:
1. Rename this import to BlogPagination
2. Rename the BlogNavigation js and scss module files

Reply:
good catch... thank you for that!